tutorials/katacoda/thanos/2-lts/step3.md

```diff
@@ -0,0 +1,73 @@
+# Step 3 - Installing the Thanos Store
+
+In this step, we will learn about Thanos Store Gateway, how to start, and what problems are solved by it.
+
+## Thanos Components
+
+Thanos is a single Go binary capable to run in different modes. Each mode represents a different component and can be invoked in a single command.
+
+Let's take a look at all the Thanos commands:
+
+```docker run --rm quay.io/thanos/thanos:v0.15.0 --help```{{execute}}
+
+You should see multiple commands that solve different purposes, block storage based long-term storage for Prometheus.
+
+In this step we will focus on thanos `store gateway`:
+
+```
+  store [<flags>]
+    Store node giving access to blocks in a bucket provider
+```
+
+## Store Gateway/ Store :
+
+* This component implements the Store API on top of historical data in an object storage bucket. It acts primarily as an API gateway and therefore does not need significant amounts of local disk space.
+* It joins a Thanos cluster on startup and advertises the data it can access.
+* It keeps a small amount of information about all remote blocks on the local disk and keeps it in sync with the bucket.
+This data is generally safe to delete across restarts at the cost of increased startup times.
+
+
+You can read more about [Store](https://thanos.io/tip/components/store.md/) here.
+
+## Deployment
+
+Click on the snippet to deploy thanos store to the running Prometheus instance.
+
+### Deploying store to "EU1" Prometheus
+
+```
+docker run -d --net=host --rm \
+    -v $(pwd)/bucket_storage.yml:/etc/prometheus/bucket_storage.yml \
+    -v $(pwd)/test:/prometheus \
+    --name thanos-store \
+    quay.io/thanos/thanos:v0.15.0 \
+    store \
+    --data-dir             /prometheus \
+    --objstore.config-file /etc/prometheus/bucket_storage.yml \
+    --http-address         0.0.0.0:10905 \
+    --grpc-address         0.0.0.0:10906 && echo "Thanos Store added"
+```{{execute}}
+
+## How to query Thanos store data?
+
+In this step, we will see how we can query Thanos store data which has access to historical data from the `thanos` bucket, and let's play with this setup a bit.
+
+Click on the [Querier UI `Graph` page](https://[[HOST_SUBDOMAIN]]-29090-[[KATACODA_HOST]].environments.katacoda.com/graph) and try querying data for a year or two by inserting metrics [k8s_app_metric0](https://[[HOST_SUBDOMAIN]]-29090-[[KATACODA_HOST]].environments.katacoda.com/graph?g0.expr=k8s_app_metric0&g0.tab=1). Make sure `deduplication` is selected and you will be able to discover all the data fetched by Thanos store.
```

Comment:
Let's maybe just use one link (second one) and link directly to metric? And just explain what is shows? (:

Reply:
Also let's make sure this links to the correct range. time and graph: e.g https://2886795301-29090-ollie08.environments.katacoda.com/graph?g0.range_input=2d&g0.end_input=2019-10-18%2000%3A00&g0.max_source_resolution=0s&g0.expr=k8s_app_metric0&g0.tab=0

Reply:
I think it's better to have both the links!

The explanation is provided below the command :)

Reply:
Link corrected, thanks !!